Automattic/wp-calypso
# CI: remove unused configurations and further tweaks to filter unnecessary builds.

## Proposed Changes

This PR is partly chore, partly enhancement to the CI.

Key changes:
- do not trigger WPComPlugin builds on any changes to:
  - `test/e2e/`
  - `packages/calypso-e2e`
- remove unused build configurations `HelpCentreE2E` and `KPIDashboardTests`.

## Testing Instructions

Ensure the following build configurations are passing:

Additionally, ensure no build has the "Failed to load settings from VCS" error.

## Pre-merge Checklist
